src/index.js.flow

```diff
@@ -1,199 +0,0 @@
-// @flow
```

Comment:
@jbaxleyiii, what's the reason behind removing Flow defs?

Reply:
What happened here :eyes: